DIY Reminders with Azure and Twilio/blog.md

```diff
@@ -14,38 +14,38 @@
 
 <img src=".\pics\03.PNG" style="zoom:75%;" />	
 
-​	When filling out the time, remember that it's in UTC and  the numbers are '{sec}{min}{hr}{day}{month}{day of week}'
+​	When filling out the time, remember that it's in UTC, and  the numbers are '{sec}{min}{hr}{day}{month}{day of week}'
 
 <img src=".\pics\04.PNG" style="zoom:75%;" />
 
-​	That's it for now. The Azure timer function is all set up and now let's integrate Twilio so we can get a notification.
+​	That's it for now. The Azure timer function is all set up. Let's integrate Twilio, so we can get a notification.
 
 ### Step 2: Setting up Twilio Credentials
-​	Firstly, let's make sure we have Twilio installed. We'll go to the left panel of our app menu in Azure, and select console. Then type: 
+​	Firstly, let's make sure we have Twilio installed. We'll go to the left panel of our app menu in Azure and select console. Then type: 
 
 ​	npm install Twilio
 
 <img src=".\pics\05.PNG" style="zoom:75%;" />
 
 ​	Now let's go over to our Twilio account and purchase a phone number. The free Twilio trial comes with some balance so we can purchase a phone number and send calls and texts using that balance.
 
-​	Back over in Azure, let's go to the same left panel as earlier and go into the Configuration tab. There let's create 4 new application settings for your TWILIO_SID, TWILIO_TOKEN, SENDER_NUMBER, RECIPIENT_NUMBER. Those are all values that we need in our code later, but for security, we put those values under application settings instead of directly writing it into our code. 
+​	Back over in Azure, let's go to the same left panel as we did earlier and go into the Configuration tab. There let's create 4 new application settings for your TWILIO_SID, TWILIO_TOKEN, SENDER_NUMBER, RECIPIENT_NUMBER. Those are all values that we need in our code later, but for security, we save those values under their corresponding name in the application settings instead of directly writing it into our code so other people can't see those credentials. 
 
 <img src=".\pics\06.PNG" style="zoom:75%;" />
 
 ### Step 3: Sending a SMS
 
-​	And now, the code. Let's go back to that left menu, select "Functions" and click on the timer trigger we made earlier. There will be a "Code + Test" and we'll be editing index.js.
+​	And now, the code. Let's go back to that left menu, select "Functions" and click on the timer trigger we made earlier. There will be a "Code + Test", and we'll be editing index.js.
 
 ​	Here's a look at the code:
 
 <img src=".\pics\08.PNG" style="zoom:75%;" />
 
-​	It's not too complex. If the timer is past due, write that in the log, then create and send a message from SENDER_NUMBER to RECIPIENT_NUMBER. If everything works perfectly, log a success, else log the error. 
+​	Let's do a quick rundown of the code. If the timer is past due, write that in the log, then create and send a message from SENDER_NUMBER to RECIPIENT_NUMBER. If everything works perfectly, log a success, else log the error. 
```

Comment:
For the last sentence in this part, the wording here isn't clear. Is it meant to convey something along the lines of: "If everything works perfectly, log a success. If not, log the error."